### PR TITLE
Add 3rd lib in app

### PR DESCRIPTION
**Pull request by li-wu**

### CODE REVIEW

File: splunk_eventgen/splunk_app/README/eventgen.conf.spec [hunk 1]
```diff
@@ -22,7 +22,7 @@
 [global]
 disabled = false
 debug = false
-verbose = false
+verbosity = false
```
Comment (li-wu) [inline splunk_eventgen/splunk_app/README/eventgen.conf.spec]: Make the spec consistent for this change: https://github.com/splunk/eventgen/pull/191

File: splunk_eventgen/lib/plugins/generator/jinja.py [hunk 2]
```diff
@@ -193,8 +193,7 @@ def gen(self, count, earliest, latest, samplename=None):
             if self._sample.splunkEmbedded is True:
                 splunk_home = os.environ["SPLUNK_HOME"]
                 app_name = getattr(self._sample, 'app', 'SA-Eventgen')
-                sample_dir = os.path.join(splunk_home, 'etc', 'apps', app_name,
-                                          'default', self._sample.DEFAULT_SAMPLE_DIR)
+                sample_dir = os.path.join(splunk_home, 'etc', 'apps', app_name, 'samples')
```
Comment (li-wu) [inline splunk_eventgen/lib/plugins/generator/jinja.py]: Fix bug introduced in this pull request: https://github.com/splunk/eventgen/pull/177

File: splunk_eventgen/__main__.py [hunk 3]
```diff
@@ -308,6 +308,13 @@ def build_splunk_app(dest, source=os.getcwd(), remove=True):
     directory_default_dir = os.path.join(directory, 'default', 'eventgen.conf')
     eventgen_conf = os.path.join(module_path, 'default', 'eventgen.conf')
     shutil.copyfile(eventgen_conf, directory_default_dir)
+
```
Comment (li-wu) [inline splunk_eventgen/__main__.py]: Add `ujson` and `jinja2` dependency when building the app.

Comment (lephino) [reply splunk_eventgen/__main__.py]: Are we sure we want to agree for the install of these libraries for the end user?

File: splunk_eventgen/__main__.py [hunk 4]
```diff
@@ -308,6 +308,13 @@ def build_splunk_app(dest, source=os.getcwd(), remove=True):
...
+    # install 3rd lib dependencies
+    install_target = os.path.join(directory, 'lib')
+    install_cmd = "pip install jinja2 ujson --upgrade --no-compile --no-binary :all: --target " + install_target
```
[the remark this answers is not in the document]
Comment (li-wu) [reply splunk_eventgen/__main__.py]: changed to use `requirements.txt`

File: splunk_eventgen/__main__.py (hunk 3 above)
Comment (li-wu) [reply splunk_eventgen/__main__.py]: Both are BSD license based and I will check it with legal.

Comment (jmeixensperger) [reply splunk_eventgen/__main__.py]: Any update here?

Comment (li-wu) [reply splunk_eventgen/__main__.py]: The process with OSS will start with all the 3rd party libs used by Eventgen.  Not just these two in app.

**Pull request merged**